docs/_theme/index.hbs

```diff
@@ -171,6 +171,9 @@ var map = new mapboxgl.Map({
 });
 {% endhighlight %}
 </div>
+
+<p class="space-top2">Also, add to your site the Mapbox GL JS CSS from <code>mapbox-gl/dist/mapbox-gl.css</code> or (via CDN) <code>{{site.tileApi}}/mapbox-gl-js/v{{site.data.package.version}}/mapbox-gl.css</code>.</p>
```

Comment:
As in https://github.com/mapbox/mapbox-gl-js/pull/4021#discussion_r96993197, I think it couldn't hurt to include `node_modules/...` in the path.